includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -926,41 +948,6 @@ public function set_payment_method_title_for_order( $order, $payment_method_type
 		$order->save();
 	}
 
-	/**
-	 * Saves the payment token to the order.
-	 *
-	 * @param WC_Order         $order The order.
-	 * @param WC_Payment_Token $token The token to save.
-	 */
-	public function add_token_to_order( $order, $token ) {
```

Comment:
Unfortunately Stripe doesn't use order tokens, it uses post meta, so I had to adapt the old code here instead.